src/browser.ts

```diff
@@ -40,15 +41,30 @@ export interface BrowserConfig {
   name: BrowserName;
   /** Whether to run in headless mode. */
   headless: boolean;
+  /** A remote WebDriver server to launch the browser from. */
+  remoteUrl: string;
```

Comment:
Recommend making remoteUrl a `string | undefined` for clarity.

Reply:
Done (actually made it fully optional, since that makes the test data easier to read too).